docs/developers/CwlParsing.md

```diff
@@ -0,0 +1,55 @@
+# How CWL is Parsed
+
+The process of parsing CWL runs approximately like this:
+
+1. Produce a canonical one-file representation of the CWL as a JSON object in memory 
+1. Use Circe to turn that representation into scala case classes
```

Comment:
ToL: When this is ready to merge, consider switching the numbers from `1. 1. 1. …` to `1. 2. 3. …`. There are a dozens of us who read md files using `less`.